### DISCUSSION TITLE
Ability to open two (or more) channels at the same time

### DISCUSSION THREAD

**Discussion by harshilsharma63** [mattermost/community#126] [Moonshot Ideas]
I often face this issue where I have to cross-reference posts from multiple channels. Manually switching between the channels feels very tiring and unoptimized.

I propose adding the ability to open two (or more) channels together, splitting the center panel vertically.

Something like this-

<img width="2048" alt="Screenshot 2021-06-25 at 12 37 46 PM" src="https://user-images.githubusercontent.com/18575143/123384543-282f1000-d5b2-11eb-8131-ede1c3342391.png">

**Comment by esethna** [comment]
Related is the idea of multiple windows or pop-out chats: https://github.com/mattermost/community/discussions/17
